gap/oper.gi

```diff
@@ -1725,6 +1725,38 @@ function(D, list)
   return DigraphShortestDistance(D, list[1], list[2]);
 end);
 
+InstallMethod(VerticesReachableFrom, "for a digraph and a vertex",
+[IsDigraph, IsPosInt],
+function(D, root)
+  local N, index, current, succ, visited, prev, n, i, parent;
+  N := DigraphNrVertices(D);
+  index := ListWithIdenticalEntries(N, 1);
+  current := root;
+  succ := OutNeighbours(D);
+  visited := [];
```

Comment:
I think `visited` is unnecessary. If `index` is initialised to all `0`, then we can check `if index[n] <> 0` in line 1743. This will use a bit less space, and will also be quicker because the check `n in visited` has complexity O(log2(n)) where n is the length of `visited` but the suggested check has constant time.

Reply:
What I wrote above isn't really right, of course, `visited` is required because it's the return value of the function. So, my suggested changes are as indicated.